ha/k8s_setup/ha_setup.py

```diff
@@ -199,9 +219,21 @@ def process(self):
             with open(const.HA_CONFIG_FILE, 'w+') as conf_file:
                 yaml.dump(conf_file_dict, conf_file, default_flow_style=False)
 
+            Cmd.copy_file(const.SOURCE_HEALTH_HIERARCHY_FILE, const.HEALTH_HIERARCHY_FILE)
             # First populate the ha.conf and then do init. Because, in the init, this file will
             # be stored in the confstore as key values
             ConfigManager.init("ha_setup")
+
+            machine_id = self.get_machine_id()
```

Comment:
Please add a comment why this machine Id is being fetched here

Reply:
Done